velox/functions/sparksql/fuzzer/CMakeLists.txt

```diff
@@ -12,15 +12,113 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Set up Spark connect
+file(
+  GLOB PROTO_FILES
+  RELATIVE ${PROJECT_SOURCE_DIR}
+  ${CMAKE_CURRENT_SOURCE_DIR}/proto/spark/connect/*.proto)
```

Comment:
Please add the `CONFIGURE_DEPENDS` flag when using glob on source files: https://cmake.org/cmake/help/latest/command/file.html#glob

Reply:
Thanks for the suggestion. Added this flag.